### PR TITLE
History of osu! - rework

### LINKED ISSUE #160: Extending and Improving History Of osu! Family of Articles
the HOO Article has been left untouched for a while. I think it's about time we extend it

### TODO : 
- [ ] Check for citations and resources that points to the 2012-2016 changes. (2015-2016's trails are still evident)
- [ ] Rewrite Article(s).

Anyone can take over this issue

### PR DESCRIPTION
**Pull request by MegaApplePi**
## Description

fix stuff then start recording for 2018

## Miscellaneous Checks

- [x] Make sure everything is in the **past tense**; this stuff already happened
- [x] Read through everything out loud and make sure it sounds right ... partially done, I began skimming around 2014 and onwards

## Article Progress

- [x] (main page) - ✔️ OK
- [x] 2007 - ✔️ OK
- [x] 2008 - ✔️ OK
- [x] 2009 - ⚠️ OK, but with holes
- [x] 2010 - ❌ created, but empty
- [x] 2011 - ❌ created, but empty
- [x] 2012 - ⚠️ OK, but with holes
- [x] 2013 - ⚠️ OK, but with holes
- [x] 2014 - ⚠️ OK, but with holes
- [x] 2015 - ⚠️ OK, but with holes
- [x] 2016 - ❌ created, but empty
- [x] 2017 - ⚠️ OK, but with holes
- [x] 2018 - ⌛️ created, will record over time
- [x] osu!wiki - ✔️ OK
- [x] Release_b335 - 🗑 Deleted
- [x] `redirect.yaml` - ✔️ updated

## Expected Redirect Changes

- The `hoo/#` redirects will redirect to their odd number years (for legacy purposes; unless it really isn't needed) ... (we should no longer be adding more of these after `hoo/6`)
  - "hoo/1" -> "History_of_osu!/2007"
  - "hoo/2" -> "History_of_osu!/2009"
  - "hoo/3" -> "History_of_osu!/2011"
  - "hoo/4" -> "History_of_osu!/2013"
  - "hoo/5" -> "History_of_osu!/2015"
  - "hoo/6" -> "History_of_osu!/2017"
- The yearly articles will get a redirect to their respected articles:
  - "hoo/2007" -> "History_of_osu!/2007"
  - "hoo/2008" -> "History_of_osu!/2008"
  - "hoo/2009" -> "History_of_osu!/2009"
  - "hoo/2010" -> "History_of_osu!/2010"
  - "hoo/2011" -> "History_of_osu!/2011"
  - "hoo/2012" -> "History_of_osu!/2012"
  - "hoo/2013" -> "History_of_osu!/2013"
  - "hoo/2014" -> "History_of_osu!/2014"
  - "hoo/2015" -> "History_of_osu!/2015"
  - "hoo/2016" -> "History_of_osu!/2016"
  - "hoo/2017" -> "History_of_osu!/2017"
  - "hoo/2018" -> "History_of_osu!/2018"
  - "hoo/osu!wiki" -> "History_of_osu!/osu!wiki"
- deleted redirects:
  - "hoo/release_b335"

## Additional Notes

closes #160

### CODE REVIEW

**Comment by RaikireHiuduo** [comment]
`osu!wiki` used to be maintained by me. osu!wiki is still part of osu! history so I still think it is worth keeping it even if it is updated like once a year or so. I am biased to keep this one for historical reason.

`Release_b335` is the forum announcement of release of Multi feature in osu!. I think at least two page (Multi and HOO/1) has a link to the page. Forum is hidden from public which I do not know why now. Can be removed since I have it copied as a proof and date and time check.

File: wiki/History_of_osu!/en.md [hunk 1]
```diff
@@ -1,10 +1,22 @@
...
-
-[History of osu!wiki can be found here instead.](/wiki/History_of_osu!/osu!wiki "osu!wiki history")
+Records of osu!'s history, all in the osu!wiki.
+
+- [2007](/wiki/HOO/2007) - Initial game build
+- [2008](/wiki/HOO/2008) - Taiko and Catch the Beat, more mods, and major improvements
+- [2009](/wiki/HOO/2009) - osu! iPhone
+- 2010
+- [2011](/wiki/HOO/2011) - osu!wiki (MediaWiki)
+- [2012](/wiki/HOO/2012) - osu! is 5, osu!mania, (some) widescreen support, and visual settings
```
Comment (TPGPL) [inline wiki/History_of_osu!/en.md]: 5th Annniversary instead of osu is 5

Comment (TPGPL) [reply wiki/History_of_osu!/en.md]: Delete some

File: wiki/History_of_osu!/en.md [hunk 2]
```diff
@@ -1,10 +1,22 @@
...
+- [2013](/wiki/HOO/2013) - UI overhaul, HD skins, (rest of) widescreen support, osu!mania editor, in-game collections, moddingV2 was introduced, Disqus, osu!api, accuracy overlays, Comic Fiesta 2013, downloads unrestricted\*, and osu!cuttingedge (osu!supporters only)
```
Comment (TPGPL) [inline wiki/History_of_osu!/en.md]: Rest of -> more

File: wiki/History_of_osu!/en.md [hunk 3]
```diff
@@ -1,10 +1,22 @@
...
+- [2014](/wiki/HOO/2014) - BPM scrolling, in-game intro/outro sequences, osu!cuttingedge (for everyone\*), and snow!
+- [2015](/wiki/HOO/2015) - jizz (redesigned osu!web), osu!weekly, and osu!coins
+- [2016](/wiki/HOO/2016) - scorev2, open-source osu! announced (osu!lazer), and osu!wiki (Markdown/Git)
+- [2017](/wiki/HOO/2017) - first osu!lazer build release, and Discord
+- [2018](/wiki/HOO/2018) - ?
```
Comment (TPGPL) [inline wiki/History_of_osu!/en.md]: Discord integration

File: wiki/History_of_osu!/en.md [hunk 4]
```diff
@@ -1,10 +1,22 @@
...
+
+---
+
+On a side note, the osu!wiki has a bit of its own detailed history in and of itself.
```
Comment (TPGPL) [inline wiki/History_of_osu!/en.md]: On the other side

File: wiki/History_of_osu!/en.md [hunk 5]
```diff
@@ -1,10 +1,22 @@
...
+- [2007](/wiki/HOO/2007) - Initial game build
+- [2008](/wiki/HOO/2008) - Taiko and Catch the Beat, more mods, and major improvements
+- [2009](/wiki/HOO/2009) - osu! iPhone
+- 2010
+- [2011](/wiki/HOO/2011) - osu!wiki (MediaWiki)
+- [2012](/wiki/HOO/2012) - osu! is 5, osu!mania, (some) widescreen support, and visual settings
+- [2013](/wiki/HOO/2013) - UI overhaul, HD skins, (rest of) widescreen support, osu!mania editor, in-game collections, moddingV2 was introduced, Disqus, osu!api, accuracy overlays, Comic Fiesta 2013, downloads unrestricted\*, and osu!cuttingedge (osu!supporters only)
+- [2014](/wiki/HOO/2014) - BPM scrolling, in-game intro/outro sequences, osu!cuttingedge (for everyone\*), and snow!
+- [2015](/wiki/HOO/2015) - jizz (redesigned osu!web), osu!weekly, and osu!coins
+- [2016](/wiki/HOO/2016) - scorev2, open-source osu! announced (osu!lazer), and osu!wiki (Markdown/Git)
```
Comment (TPGPL) [inline wiki/History_of_osu!/en.md]: and osu!wiki overhaul

File: wiki/History_of_osu!/osu!wiki/en.md [hunk 6]
```diff
@@ -1,31 +1,31 @@
-# osu!wiki history
+# osu!wiki
 
 ## MediaWiki (2011 - 2017)
```
Comment (TPGPL) [inline wiki/History_of_osu!/osu!wiki/en.md]: Not 2011 - 2016?

File: wiki/History_of_osu!/osu!wiki/en.md [hunk 7]
```diff
@@ -1,31 +1,31 @@
...
-2014 | osu!wiki becomes an actual information centre of osu!, replacing most of the web copies such as [The Team](/wiki/People/The_Team) and [Rules](/wiki/Rules). Also, osu!wiki now has 5 admins (Loctav, deadbeat).
-2015 | osu!wiki received major translation updates from other languages. Dellirium has been replaced by Shiro as one of the admins of osu!wiki.
-22 February 2016 | osu!wiki now has 8 admins (Ukami and Galkan).
-01 April 2016 | osu!wiki now has 9 admins (Nathanael).
-30 August 2016 | osu!wiki running on MediaWiki is being deprecated for GitHub repository version. The current osu!wiki will stay until the GitHub version is ready to be deployed with all the pages and images ported.
-June 2017 | osu!wiki running on MediaWiki is officially deprecated. The link now points to the new osu-web interface using the osu!wiki resources made in the GitHub repository. [A temporary backup of the osu!wiki without the MediaWiki functionality can be found here.](https://osu.ppy.sh/old-wiki/Main_Page)
+| Date, Time | Description |
+| --- | --- |
+| 2011-12-05T08:13Z | First edit of osu!wiki by Ephemeral. |
```
Comment (TPGPL) [inline wiki/History_of_osu!/osu!wiki/en.md]: 13Z?

File: wiki/History_of_osu!/osu!wiki/en.md [hunk 8]
```diff
@@ -1,31 +1,31 @@
...
+| 2011-12-06 | [osu!wiki is now live!](https://osu.ppy.sh/forum/t/68525). |
+| 2012-11 | osu!wiki now has 3 admins (Dellirium, Repflez). Previously, only peppy was the admin. |
+| 2013-01-27T06:12 | peppy integrated the osu!wiki with the main site design. [Link to peppy post](http://osu.ppy.sh/forum/t/68479/start=124). |
```
Comment (TPGPL) [inline wiki/History_of_osu!/osu!wiki/en.md]: peppy's

File: wiki/History_of_osu!/osu!wiki/en.md [hunk 9]
```diff
@@ -1,31 +1,31 @@
...
+| 2014 | osu!wiki becomes an actual information centre of osu!, replacing most of the web copies such as [The Team](/wiki/People/The_Team) and [Rules](/wiki/Rules). Also, osu!wiki now has 5 admins (Loctav, deadbeat). |
+| 2015 | osu!wiki received major translation updates from other languages. Dellirium has been replaced by Shiro as one of the admins of osu!wiki. |
+| 2016-02-22 | osu!wiki now has 8 admins (Ukami and Galkan). |
+| 2016-04-01 | osu!wiki now has 9 admins (Nathanael). |
+| 2016-08-30 | osu!wiki running on MediaWiki is being deprecated for GitHub repository version. The current osu!wiki will stay until the GitHub version is ready to be deployed with all the pages and images ported. |
```
Comment (TPGPL) [inline wiki/History_of_osu!/osu!wiki/en.md]: For -> in favour of

File: wiki/History_of_osu!/osu!wiki/en.md [hunk 10]
```diff
@@ -1,31 +1,31 @@
...
-30 May 2017 | [Page redirection is finally implemented in osu!wiki](https://github.com/ppy/osu-web/pull/1144 "osu!wiki - redirects by MegaApplePi")
-June 2017 | TPGPL is promoted with write-access for the osu!wiki repository. Search service on osu!web for osu!wiki is implemented.
\ No newline at end of file
+| Date | Description |
+| --- | --- |
+| 2016-08-26 | [peppy created osu-wiki repository](https://github.com/ppy/osu-wiki/tree/3433cbeeda9303a470647cad1c338d43f4272a2e "osu-wiki repository"). craftu and Xyloo ported the MediaWiki content to the repository in a week. |
+| 2016-09 | Most of the pages except the images are moved to the repository, with unsolved syntax-translation errors stemming from language difference between Markdown and MediaWiki. |
+| 2017-01 | osu!web support of osu!wiki done by nanaya. |
+| 2017-01-05 | [osu!wiki project suggested by Ephemeralis](https://github.com/ppy/osu-wiki/projects/1 "osu!wiki 2017 project by Ephemeralis"). |
+| 2017-01-26 | osu!wiki got mentioned on [ppy blog](http://pppy.tumblr.com/post/156390386433/2017-01-dev-meeting "ppy blog - 2017-01 DEV MEETING"). |
```
Comment (TPGPL) [inline wiki/History_of_osu!/osu!wiki/en.md]: ppy's

File: wiki/History_of_osu!/en.md (hunk 4 above)
Comment (MegaApplePi) [reply wiki/History_of_osu!/en.md]: osu!wiki isn't really the other side of osu!, I had meant for it to be a small thing on its own.

File: wiki/History_of_osu!/osu!wiki/en.md (hunk 7 above)
Comment (MegaApplePi) [reply wiki/History_of_osu!/osu!wiki/en.md]: Z = +00:00
its part of ISO 8601

File: wiki/History_of_osu!/osu!wiki/en.md (hunk 6 above)
Comment (MegaApplePi) [reply wiki/History_of_osu!/osu!wiki/en.md]: There was a fading transition time where MediaWiki and GitHub co-existed at the same time. The one that should be fixed is the other heading.

File: wiki/History_of_osu!/osu!wiki/en.md (hunk 10 above)
Comment (MegaApplePi) [reply wiki/History_of_osu!/osu!wiki/en.md]: strange, I can't view that link... will remove link

File: wiki/History_of_osu!/en.md (hunk 1 above)
Comment (TPGPL) [reply wiki/History_of_osu!/en.md]: Prob

File: wiki/History_of_osu!/osu!wiki/en.md [hunk 11]
```diff
@@ -1,31 +1,31 @@
...
-June 2017 | TPGPL is promoted with write-access for the osu!wiki repository. Search service on osu!web for osu!wiki is implemented.
\ No newline at end of file
+| Date | Description |
+| --- | --- |
+| 2016-08-26 | [peppy created osu-wiki repository](https://github.com/ppy/osu-wiki/tree/3433cbeeda9303a470647cad1c338d43f4272a2e "osu-wiki repository"). craftu and Xyloo ported the MediaWiki content to the repository in a week. |
+| 2016-09 | Most of the pages except the images are moved to the repository, with unsolved syntax-translation errors stemming from language differences between Markdown and MediaWiki. |
+| 2017-01 | osu!web support of osu!wiki done by nanaya. |
+| 2017-01-05 | [osu!wiki project suggested by Ephemeralis](https://github.com/ppy/osu-wiki/projects/1 "osu!wiki 2017 project by Ephemeralis"). |
+| 2017-01-26 | osu!wiki got mentioned on ppy's blog. |
+| 2017-05-22 | [osu!news archive is now stored on osu!wiki (in text form only)](https://github.com/ppy/osu-wiki/pull/252 "Initial commit of all tumblr news posts by peppy"). |
```
Comment (TPGPL) [inline wiki/History_of_osu!/osu!wiki/en.md]: osu!news are now stored

File: wiki/History_of_osu!/2017/en.md [hunk 12]
```diff
@@ -0,0 +1,30 @@
...
+The osu!team participates in Anime Expo for the upcoming 10 years of osu!.
+
+Links:
+
+- [2017.609.0 – ppy blog](https://blog.ppy.sh/2017.609.0/)
+- [2017.717.0 – ppy blog](https://blog.ppy.sh/2017.717.0/)
+
+## December
+
+OWC 2017 concludes with Poland being victorious. Another handful of community contributors were announced. Thanks to LiquidPL, the cringey download page in osu!web has been redone. Every player was given osu!supporter status temporarily for testing purposes, which coincidentally happened on Christmas Eve.
```
Comment (TPGPL) [inline wiki/History_of_osu!/2017/en.md]: was given -> received/obtained

File: wiki/History_of_osu!/2015/en.md [hunk 13]
```diff
@@ -0,0 +1,33 @@
...
+The next day, peppy decides to rollback on the osu!coins due to popular demand. A new subdomain for the osu!web redesign was opened for preview.
+
+Links;
+
+- [peppy's remarks on osu!coins](https://disq.us/p/w2ydos)
+- [osu!next](http://osunext.tumblr.com/post/115384460628/the-super-early-ultra-alpha-buggy-read-only-of-the)
+
+## July
+
+peppy and flyte briefly explains why the temporary osu!web redesign subdomain was `jizz.ppy.sh` in an episode of osu!talk.
```
Comment (TPGPL) [inline wiki/History_of_osu!/2015/en.md]: explain

File: wiki/History_of_osu!/2014/en.md [hunk 14]
```diff
@@ -0,0 +1,27 @@
+# 2014
+
+*Main page: [History of osu!](/wiki/HOO).*
+
+## March
+
+As skinning elements expanded, the element, `playfield.jpg` was removed in it's line of duty and replaced with a new generic one that was unskinnable. `playfield.jpg` was an element that allowed you to use any image as the default background of a beatmap, if one isn't supplied.
```
Comment (TPGPL) [inline wiki/History_of_osu!/2014/en.md]: new generic one that **is** unskinnable

File: wiki/History_of_osu!/osu!wiki/en.md (hunk 11 above)
Comment (MegaApplePi) [reply wiki/History_of_osu!/osu!wiki/en.md]: osu!news is one thing tho...?

File: wiki/History_of_osu!/2014/en.md (hunk 14 above)
Comment (MegaApplePi) [reply wiki/History_of_osu!/2014/en.md]: can you explain why it should be `is` instead of `was`?

Comment (TPGPL) [reply wiki/History_of_osu!/2014/en.md]: because it's still unskinable, it didnt change

Comment (MegaApplePi) [reply wiki/History_of_osu!/2014/en.md]: I will change it, but I don't fully agree to it :s

File: wiki/History_of_osu!/2015/en.md (hunk 13 above)
Comment (MegaApplePi) [reply wiki/History_of_osu!/2015/en.md]: `explained`, I was watching the video when I added that, so I mistook it as "happening now"

File: wiki/History_of_osu!/2007/en.md [hunk 15]
```diff
@@ -0,0 +1,99 @@
...
+
+## December
+
+**Songs can be edited by dragging them into the osu! window. Tooltips were now present explaining all the various buttons in the osu! menus and editor interface.** Many new quality skins were submitted by early skinners ([Holiday by LuigiHann](http://osu.ppy.sh/forum/t/1139/start=0), [WindWaker by awp](http://osu.ppy.sh/forum/t/761/start=0)). **New sorting methods were added to the song selection screen**. **peppy could now send universal announcements to all osu! players. Minor tweaks were made to gameplay, such as non-active circle dimming.**
```
Comment (TPGPL) [inline wiki/History_of_osu!/2007/en.md]: can be -> could be

Comment (TPGPL) [inline wiki/History_of_osu!/2007/en.md]: delete now in the second sentence

**Pull request merged**